frontend/src/components/forecast/ForecastCity.tsx

```diff
@@ -1,18 +1,40 @@
 import ForecastDayCard from "./ForecastDayCard";
+import { useState } from "react";
 
 const ForecastCity: React.FC = () => {
+  const [activeCard, setActiveCard] = useState(0);
   const cityName = "London";
 
+  const updateActiveCard = (index: number) => {
+    setActiveCard(index);
+  };
+
   return (
     <>
       <div className="mt-40">
         <h3 className="city">{cityName}</h3>
         <div className="forecast-day-container">
-          <ForecastDayCard />
-          <ForecastDayCard />
-          <ForecastDayCard />
-          <ForecastDayCard />
-          <ForecastDayCard />
+          {/* hardcoded index */}
+          <ForecastDayCard
+            isActive={0 === activeCard}
+            toggleActive={() => updateActiveCard(0)}
+          />
+          <ForecastDayCard
+            isActive={1 === activeCard}
+            toggleActive={() => updateActiveCard(1)}
+          />
+          <ForecastDayCard
+            isActive={2 === activeCard}
+            toggleActive={() => updateActiveCard(2)}
+          />
+          <ForecastDayCard
+            isActive={3 === activeCard}
+            toggleActive={() => updateActiveCard(3)}
+          />
+          <ForecastDayCard
+            isActive={4 === activeCard}
+            toggleActive={() => updateActiveCard(4)}
+          />
```

Comment:
This would be much better as a loop to avoid the duplication, something like 

```
for(let days = 0; days > 5; days++) {
  <ForecastDayCard
      isActive={days === activeCard}
      toggleActive={() => updateActiveCard(days)}
      />
 }
 ```
 
Something like that should work, may need some tweaks.
 For loop could be 'days > 5' or 'days < 5', not 100% sure.